lib/bike_brigade_web/live/itinerary_live/index.html.heex

```diff
@@ -45,57 +45,51 @@
   </div>
 </nav>
 <%= if @campaign_riders != [] do %>
-  <%= for cr <- @campaign_riders do %>
-    <div class="flex w-full bg-white shadow sm:my-1 sm:rounded-md">
-      <ul role="list" class="w-full divide-y divide-gray-200">
-        <li id={"campaign-#{cr.campaign.id}"}>
-          <div class="px-4 py-4">
-            <div class="items-center justify-between md:flex">
-              <div class="flex items-center mb-2 space-x-1">
-                <.link
-                  navigate={~p"/campaigns/#{cr.campaign}"}
-                  class="font-bold link"
-                >
-                  <%= name(cr.campaign) %>
-                </.link>
+  <div class="flex flex-col mt-8">
+    <%= for cr <- @campaign_riders do %>
```

Comment:
Might as well switch to the new heex syntax while we're here

Reply:
Welp, that shrunk the PR 🥳